mcstatus/pinger.py

```diff
@@ -139,8 +139,18 @@ def __init__(self, raw):
 
         if "description" not in raw:
             raise ValueError("Invalid status object (no 'description' value)")
-        self.description = raw["description"]
-        self.description_clean = re.sub(r'\u00A7.', '', raw["description"])
+        # for newer versions of minecraft
+
+        if type(raw["description"]) is str:
```

Comment:
Switching to `isinstance()` checks may be better, and using `string_types` instead of `str` would be more version compatible